test/release/examples/primers/modules.chpl

```diff
@@ -407,7 +444,9 @@ module MainModule {
 
     /* Remember: this is in contrast to ``import`` statements, where only the
        module name or specific symbols within the module are brought in, rather
-       than both.
+       than both.  Also remember that you can write a ``use`` statement that
+       doesn't enable references to the module's name, by renaming the module
```

Comment:
Same feedback in the primer about "references". I like "mentions" or "accesses".